script/tasks/diamondUpdatePeriphery.sh

```
@@ -179,17 +179,45 @@ register() {
+        echoDebug "Calldata: $CALLDATA"
+
+        DIAMOND_ADDRESS=$(getContractAddressFromDeploymentLogs "$NETWORK" "$ENVIRONMENT" "$DIAMOND_CONTRACT_NAME")
+        echoDebug "DIAMOND_ADDRESS: $DIAMOND_ADDRESS"
+        echoDebug "NETWORK: $NETWORK"
+
+        echo "Now proposing registerPeripheryContract("$CONTRACT_NAME","$ADDR") to diamond "$DIAMOND_ADDRESS" with calldata $CALLDATA"
+        $(ts-node script/deploy/safe/propose-to-safe.ts --to "$DIAMOND_ADDRESS" --calldata "$CALLDATA" --network "$NETWORK" --rpcUrl "$RPC_URL" --privateKey "$SAFE_SIGNER_PRIVATE_KEY")
+
+      else
+        # just register the diamond (no multisig required)
+        cast send "$DIAMOND" 'registerPeripheryContract(string,address)' "$CONTRACT_NAME" "$ADDR" --private-key $(getPrivateKey "$NETWORK" "$ENVIRONMENT") --rpc-url "$RPC_URL" --legacy >/dev/null 2>&1
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)